newrelic.yml

```diff
@@ -853,6 +853,55 @@ common: &default_settings
   # Foundry environment.
   # utilization.detect_pcf: true
 
+  #
```

Comment:
@hannahramadan we'll likely need some help here getting these comments moved into `default_source.rb` and/or the `newrelic.yml` auto-generator so that these changes aren't lost the next time the file is auto-generated.

Reply:
@fallwith roger that! I'll make a ticket for this

Reply:
Great suggestion, @fallwith. Thank you for making the ticket, @hannahramadan! I think we may also want to consider adding this content to the auto-generated configuration docs that get published on the New Relic Docs website.